index.Rmd

```diff
@@ -1,54 +1,107 @@
 
-You can do bullets like this:
+#### Analysisng CO and NO~2~ data (.mat) files in R 
 
-* The first most important thing
-* The second most important thing
-* The third most important thing
+The .mat files read in R were further filtered based on quality index. Once this is done, the data is converted to ordinary datetime format from matlab datetime format using the "lubridate" package and ymd() function. This makes the data into a readable format. 
 
-You can do numbers like this:
+#### Analysisng MODIS fire data 
 
-1. The first most important thing
-2. The second most important thing
-3. The third most important thing
+The MODIS fire data are in the form of csv files, once these files are read into R, they are in the form of data frames, the number of times a brightness value occurs in a day is taken as a fire count for that day. Number of fire counts on each day are obtained. Based on these fire counts I have filtered the study dates and would be further focusing on them. 
 
-See [http://rmarkdown.rstudio.com/](http://rmarkdown.rstudio.com/) for all the amazing things you can do.
+#### Analysing GEOS-CF model data for NO~2~  and CO 
 
+GEOS-CF model data are in the form of .h5 files, the data in these files is in the form of a meshgrid/ matrix. Longitude and Latitude values  are distributed in the form of a grid, and each point in the grid (longitude,latitude) pair has a concentration value. 
+GEOS-CF model data on specific dates is chosen based on the MODIS data, the days on which MODIS fire counts were found to be significant were taken into consideration. 
+
+#### Data Representation 
+
+Data for TROPOMI CO and NO~2 are represented in the from of maps with longitudes varying on x-axis and latitudes varying on y-axis. GEOS-CF model data is also represented in the same way. 
+
+The following packages were used in the project:
+
+```{r, message=F, warning=F}
+library(pracma)
 
-Here's my first code chunk.
-```{r}
-1+2
 ```
 
-Load any required packages in a code chunk (you may need to install some packages):
 
 ```{r, message=F, warning=F}
+library(dplyr)
+library(R.matlab)
+library(lubridate)
+library(ncdf4)
+library(ggplot2)
+library(devtools)
+library(Bilinear)
+library(interp)  
+```
+
+
+```{r, message=F, warning=F}
+library(R.matlab)
 library(tidyverse)
 library(leaflet)
 library(kableExtra)
-knitr::opts_chunk$set(cache=TRUE)  # cache the results for quick compiling
+knitr::opts_chunk$set(cache=TRUE) 
 ```
 
+# DATA HANDLING 
+#### Reading modis fire data 
+```{r , results='asis'}
+
+modis_data <- read.csv('C:\\Users\\shrut\\Desktop\\SDS Project\\fire_archive_M6_164945.csv')
```

Comment:
you'll need to move this file into the repository so others can access it.  Then load it with `read.csv("data/fire_archive..")`